crates/wasm_interp/src/value_stack.rs

```diff
@@ -69,6 +75,14 @@ impl<'a> ValueStack<'a> {
         value
     }
 
+    pub fn peek(&self) -> Value {
+        let is_64 = self.is_64[self.is_64.len() - 1];
+        let is_float = self.is_float[self.is_float.len() - 1];
```

Comment:
I just checked again and now I remember why I did this!
There is indeed a `.last()` but it gives you `Option<BitRef<Const, T, O>>` instead of just `Option<bool>`
I was having trouble getting a `bool` out of that... but I just realised you can do it with a `*` dereference.